src/Utilities/ErrorHandling/AbortWithErrorMessage.hpp

```diff
@@ -26,6 +26,7 @@
 /// We try to demangle and format the backtrace. Long symbol names are
 /// abbreviated, unless you set the `SPECTRE_SHOW_FULL_BACKTRACE_SYMBOLS`
 /// environment variable to a non-empty value (e.g. "1").
+template <typename ExceptionTypeToThrow>
```

Comment:
Can't reach with Github comments, but maybe document on `abort_with_error_message_no_trace` that it always throws a `SpectreError` even though it won't actually print the exception type?